VisualStudio.gitignore

```diff
@@ -259,15 +259,20 @@ ServiceFabricBackup/
 *.ldf
 *.ndf
 
-# Business Intelligence projects
+# SQL Server Integration Services
+*.dtproj.user
+
+# SQL Server Reporting Services
 *.rdl.data
-*.bim.layout
-*.bim_*.settings
 *.rptproj.rsuser
 *- [Bb]ackup.rdl
 *- [Bb]ackup ([0-9]).rdl
 *- [Bb]ackup ([0-9][0-9]).rdl
 
+# SQL Server Analysis Services
```

Comment:
👍 to extracting this to it's own section (if it makes sense, I'm not familiar with SSAS)

Reply:
Yes, SSRS, SSIS, and SSAS should have it's own section.